torch/csrc/jit/codegen/cuda/ir_base_nodes.h

```diff
@@ -165,10 +165,6 @@ class TORCH_CUDA_API Statement : public NonCopyable, public PolymorphicBase {
  */
 class TORCH_CUDA_API Val : public Statement {
  public:
-  virtual ~Val() = default;
-
-  Val() = delete;
```

Comment:
We get the virtual destructor from Statement (actually from PolymorphicBase). And the default constructor is implicitly deleted since we have other user-declared constructors.